src/lexer/scanner_test.go

```diff
@@ -0,0 +1,513 @@
+package lexer
+
+import (
+	"io"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func fillSymbolTable(table *SymbolTable) {
+	for _, languageToken := range LanguageReservedTokens {
+		table.Insert(languageToken.GetLexem(), languageToken)
+	}
+}
```

Comment:
I was thinking, we use this function both in `main.go` and here, so maybe it should be good put it on `utils.go`

Reply:
Think so too. I am going to add this function to the utils module.

Reply:
Changes done.